app/dashboard/src/providers/BackendProvider.tsx

```diff
@@ -61,9 +64,10 @@ export function useRemoteBackend() {
 // === useRemoteBackendStrict ===
 // ==============================
 
-/** Get the Remote Backend.
- * @throws {Error} when no Remote Backend exists. This should only happen if the user is not logged
- * in. */
+/**
+ * Get the Remote Backend.
+ * @throws {Error} when no Remote Backend exists. This should never happen.
+ */
 export function useRemoteBackendStrict() {
```

Comment:
i guess this means we should gradually phase out `useRemoteBackendStrict` (or just remove all usages immediately here)